src/components/MyMap/index.js

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState } from "react";
+import {
+  MapContainer,
+  TileLayer,
+  Marker,
+  Popup,
+  useMapEvent,
+} from "react-leaflet";
+
+const zoom = 13;
+
+function LocationMarker({ center, name, temp, fetchWeatherUsingCoordinates }) {
+  const [position, setPosition] = useState(center);
+
+  const map = useMapEvent("click", (e) => {
+    // console.log({ e, map });
+    setPosition(e.latlng);
+
+    fetchWeatherUsingCoordinates(e.latlng);
+  });
+
+  return position === null ? null : (
+    <Marker position={position}>
+      <Popup>
+        <div className="MyMap-popup-container">
+          <h2>{name}</h2>
+          <h3>{temp}°C</h3>
+        </div>
+      </Popup>
```

Comment:
This popup isn't visible to me. Can you please share a screenshot of where it is?

Reply:
The popup appears when we click on the marker, looks like this

![image](https://user-images.githubusercontent.com/56843437/142365846-38837627-9441-4c7a-9f11-1b38706fc98e.png)

Reply:
Okay got it. This looks great